third-party/cmake/BuildLuarocks.cmake

```diff
@@ -150,12 +150,12 @@ if(USE_BUNDLED_BUSTED)
   endif()
 
   # DEPENDS on the previous module, because Luarocks breaks if parallel.
-  add_custom_command(OUTPUT ${HOSTDEPS_LIB_DIR}/luarocks/rocks/penlight/1.3.2-2
+  add_custom_command(OUTPUT ${HOSTDEPS_LIB_DIR}/luarocks/rocks/penlight
     COMMAND ${LUAROCKS_BINARY}
-    ARGS build penlight 1.3.2-2 ${LUAROCKS_BUILDARGS}
+    ARGS build penlight ${LUAROCKS_BUILDARGS}
```

Comment:
I did not mean to suggest we shouldn't pin to specific versions, only that we should not reference the literal rockspec file. For penlight we're already using the luarocks command so no changes are needed.

Reply:
The only reason (1d4506ee172d90d2a931c019dcd8ecca3ed151d1) we pinned to this version was to work around a bug, which has since been fixed.